NotesApi/V2/Infrastructure/NoteDb.cs

```diff
@@ -27,5 +29,7 @@ public class NoteDb
 
         [DynamoDBProperty(Converter = typeof(DynamoDbObjectConverter<AuthorDetails>))]
         public AuthorDetails Author { get; set; }
+
+        public bool Highlight { get; set; }
```

Comment:
I would add the `[DynamoDBProperty(Converter = typeof(DynamoDbBoolConverter))]` attribute on this, otherwise the value gets persisted to the db as an `int` (i.e. `0` or `1`, rather than `true` or `false`)

Reply:
Done!